data/json/items/ammo/4570.json

```diff
@@ -78,7 +78,7 @@
     "//": "***Add blackpowder effects once #30944 gets merged***",
     "copy-from": "4570_low",
     "type": "AMMO",
-    "name": "reloaded .45-70 black powder",
+    "name": ".45-70, blackpowder",
```

Comment:
Missing reloaded.

Reply:
It was established that `blackpowder` already implies it's reloaded, so I don't see a reason to add that here (the original name was wrong to this end, anyway).